systemservices/ethwallet/mesg.yml

```diff
@@ -148,8 +145,24 @@ tasks:
         name: "Success"
         description: "Output when the task executes successfully."
         data:
-          signedTransaction:
-            name: "Signed transaction"
-            description: "The data of the signed transaction."
-            type: Object
+          raw:
```

Comment:
raw is a strange name, maybe `signedTransaction` will be more explicit

Reply:
Done

Reply:
@antho1404 review please